src/r-analysis/docker-compose.yaml

```diff
@@ -6,7 +6,7 @@ services:
     restart: always
     volumes:
       - .:/workspace:cached
-      - work:/home/rstudio:cached
+      - work:/home/r-analysis:cached
```

Comment:
did you test this? i think you need to at least override the USER env var

Reply:
nope, putting this up as a draft for now

Reply:
The app creation works well now, I decided to leave the user + home directory as is since the rstudio app probably makes a lot of assumptions around that.

Not sure why the cirrocumulus test is failing though. It says the post startup is failing.

Reply:
unlikely related to your change. if it's blocking you, can you file a ticket and disable cirro test?